src/components/editor/Store/types.ts

```diff
@@ -39,6 +40,15 @@ export interface EditorStoreState<T> {
     status: EditorStatus;
     setStatus: (newVal: EditorStatus) => void;
 
+    // Draft Initialization
+    draftInitializationError: null | {
```

Comment:
This is _really_ an alert configuration. A similar property exists in the bindings editor store and I replaced the instance of _Error_ in the name to _Alert_, which read a little better in code but it still feels odd and verges on too long. Thoughts?

Reply:
I don't think the props in stores can really be "too long" for the most part. It is interesting to consider how much the store knows about how it is getting used (alert vs error)... I don't have any strong opinions as of now.